lib/ASTGen/Sources/ASTGen/Patterns.swift

```diff
@@ -45,6 +50,60 @@ extension ASTGenVisitor {
       loc: nameLoc
     )
   }
+
+  func generate(isTypePattern node: IsTypePatternSyntax) -> BridgedIsPattern {
+    return .createParsed(
+      self.ctx,
+      isLoc: self.generateSourceLoc(node.isKeyword),
+      typeExpr: .createParsed(
+        self.ctx,
+        type: self.generate(type: node.type)
+      )
```

Comment:
Don't we have a `generate` function for `TypeExpr`?

Reply:
We do, but that only accept `TypeExprSyntax` instead of `TypeSyntax`.